packages/plugin/webpack/src/WebpackConfig.ts

```diff
@@ -194,25 +194,32 @@ export default class WebpackConfigGenerator {
           target: this.rendererTarget(entryPoint),
           devtool: this.rendererSourceMapOption,
           mode: this.mode,
-          output: {
+          output: Object.assign(entryPoint.output || {}, {
             path: path.resolve(this.webpackDir, 'renderer'),
             filename: '[name]/index.js',
             globalObject: 'self',
             ...(this.isProd ? {} : { publicPath: '/' }),
-          },
+          }),
           node: {
             __dirname: false,
             __filename: false,
           },
           plugins: [
+            ...(entryPoint.htmlPlugins || []),
             ...(entryPoint.html
               ? [
-                  new HtmlWebpackPlugin({
-                    title: entryPoint.name,
-                    template: entryPoint.html,
-                    filename: `${entryPoint.name}/index.html`,
-                    chunks: [entryPoint.name].concat(entryPoint.additionalChunks || []),
-                  }) as WebpackPluginInstance,
+                  new HtmlWebpackPlugin(
+                    Object.assign(
```

Comment:
Same here, please use the spread operator

Reply:
roger